data/glsk-file/src/main/java/com/farao_community/farao/commons/data/glsk_file/UcteGlskDocument.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.commons.data.glsk_file;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Ucte type GLSK document object: contains a list of GlskPoint
+ * @author Pengbo Wang {@literal <pengbo.wang@rte-international.com>}
+ */
+public class UcteGlskDocument {
+    /**
+     * list of GlskPoint in the give Glsk document
+     */
+    private List<GlskPoint> listUcteGlskBlocks;
+    /**
+     * map of time series id and time series
+     */
+    private Map<String, UcteGlskSeries> ucteGlskSeriesByID; //map<timeSerieID, UcteGlskSeries>
+    /**
+     * List of Glsk point by country code
+     */
+    private Map<String, List<GlskPoint>> ucteGlskPointsByCountry; //map <CountryID, List<GlskPoint>>
+
+    /**
+     * @param data input file as input stream
+     * @throws ParserConfigurationException
+     * @throws IOException
+     * @throws SAXException
+     */
+    public UcteGlskDocument(InputStream data) throws ParserConfigurationException, IOException, SAXException {
+        DocumentBuilderFactory documentBuilderFactory = DocumentBuilderFactory.newInstance();
+        documentBuilderFactory.setNamespaceAware(true);
```

Comment:
Add
`documentBuilderFactory.setAttribute(XMLConstants.FEATURE_SECURE_PROCESSING, Boolean.TRUE);
`
It should remove the Sonar error

Reply:
Try this solution on commit 34b62e9 but it does not solve the SonarCloud warning. Could be a bug of SonarCloud's rule?